README.md

```diff
@@ -89,7 +91,7 @@ const clientOnlyStore = createStore(
 );
 ```
 
-Often, you'll want to update state or trigger side effects after loading the initial URL. To maintain compatibility with other store enhancers (particularly ones that handle side effects, like `redux-loop` or `redux-saga`), we require this optional initial dispatch to happen in userland code by doing the following:
+Often, you'll want to update state or trigger side effects after loading the initial URL. To maintain compatibility with other store enhancers (particularly ones that handle side effects, like `redux-loop` or `redux-saga`), we require this optional initial dispatch to happen in client code by doing the following:
```

Comment:
I think `userland` is more accurate (as opposed to "library" land), but `client` is more obvious 👍